src/js/angular/controllers.js

```diff
@@ -652,6 +659,29 @@ function mainCtrl($scope, $menuItems, $jwtAuth, $http, toastr, $location, $repos
         }
     };
 
+    $scope.isPopoverOpen = false;
```

Comment:
It would be better to rename this property and all related functions to more specific names for better readability. The name isPopoverOpen could be associated with multiple popover components on the page. A name like isActiveRepositoryPopover,Open or something similar, would improve code readability.

Reply:
I renamed the functions and this field for better readability